src/widget/wcolorpickeraction.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <QHBoxLayout>
+#include <QWidget>
+#include <QWidgetAction>
+
+#include "util/parented_ptr.h"
+#include "widget/wcolorpicker.h"
+
+class WColorPickerAction : public QWidgetAction {
+    Q_OBJECT
+  public:
+    explicit WColorPickerAction(
+            ColorPickerOption option = ColorPickerOption::DenyNoColor,
```

Comment:
- When instantiating WColorPickerAction we should be explicit about the color option. A default value for this argument doesn't really help.
- "option" -> "colorOption"? Because their might be options for other aspects of this control, although "color" is part of the class name.